src/main/Apis/Transactions/ITransactionsApi.cs

```diff
@@ -7,6 +7,7 @@ public interface ITransactionsApi
         TransactionInitializeResponse Initialize(string email, int amount, string reference = null, bool makeReferenceUnique = false);
         TransactionInitializeResponse Initialize(TransactionInitializeRequest request, bool makeReferenceUnique = false);
         TransactionVerifyResponse Verify(string reference);
+        ChargeAuthorizationResponse ChargeAuthorization(ChargeAuthorizationRequest request);
```

Comment:
Since, this appears to be a commonly used method, it'll be more "wrist friendly" to provide an overload that accepts the `required` parameters of the `/charge_authorization` endpoint. i.e. { `authorization_code`, `email`, and `amount` }.

Reply:
Fixed at [dff94da](https://github.com/adebisi-fa/paystack-dotnet/commit/dff94da738790bb1d04b566e819e980396a55600).